kafka/README.md

```diff
@@ -1,20 +1,22 @@
 # Information for confluent-kafka-go developers
 
+Since the package does not currently use modules, export
```

Comment:
```markdown
*Go >=1.16*: Since the package ..., set the environment variable `GO111MODULE=off` or prepend it to all go commands.
```

And then remove the exports below and just use prepend, since export has side-effects.

Reply:
Done